sklearn/metrics/regression.py

```diff
@@ -355,12 +370,26 @@ def median_absolute_error(y_true, y_pred):
     >>> y_pred = [2.5, 0.0, 2, 8]
     >>> median_absolute_error(y_true, y_pred)
     0.5
+    >>> y_true = [[0.5, 1], [-1, 1], [7, -6]]
+    >>> y_pred = [[0, 2], [-1, 2], [8, -5]]
+    >>> median_absolute_error(y_true, y_pred)
+    0.75
+    >>> median_absolute_error(y_true, y_pred, multioutput='raw_values')
+    array([0.5, 1. ])
+    >>> median_absolute_error(y_true, y_pred, multioutput=[0.3, 0.7])
+    0.85
 
     """
-    y_type, y_true, y_pred, _ = _check_reg_targets(y_true, y_pred, None)
-    if y_type == 'continuous-multioutput':
-        raise ValueError("Multioutput not supported in median_absolute_error")
-    return np.median(np.abs(y_pred - y_true))
+    _, y_true, y_pred, _ = _check_reg_targets(y_true, y_pred, multioutput)
```

Comment:
Let's use ``y_type, y_true, y_pred, multioutput = _check_reg_targets(y_true, y_pred, multioutput)`` here. This will be more consistent. (Actually since we call check_array in _check_reg_targets, multioutput will sometimes be modified.)

Reply:
Good point, thanks. Now fixed.